cmd/stackdriver-prometheus-sidecar/statusz-tmpl.html

```diff
@@ -69,7 +69,7 @@ <h1>Parsed configuration</h1>
       <tr><th>Kubernetes labels: cluster name</th><td>{{.KubernetesLabels.ClusterName}}</td></tr>
       <tr><th>Kubernetes labels: location</th><td>{{.KubernetesLabels.Location}}</td></tr>
       <tr><th>Listen address</th><td>{{.ListenAddress}}</td></tr>
-      <tr><th>Log level</th><td>{{.LogLevel.String}}</td></tr>
+      <tr><th>Log config</th><td>{{.PromlogConfig}}</td></tr>
```

Comment:
If following convention as [Line 69-70](https://github.com/Stackdriver/stackdriver-prometheus-sidecar/pull/143/files#diff-368eb7f4c7270a416b59030915bab4e0L69-L70), we'd like to have two lines here - one for Log Level, one for log format.
```
<tr><th>Log level</th><td>{{.PromlogConfig.Level}}</td></tr>
<tr><th>Log format</th><td>{{.PromlogConfig.Format}}</td></tr>
```

Reply:
Yeah, I debated that. We don't own the `LogConfig` type, so I'd rather not inspect its guts. What do you think?

Reply:
this template only shows value without the key in the struct. Here is the test:
https://github.com/Stackdriver/stackdriver-prometheus-sidecar/pull/143/files#diff-dbcbb428be48c2fa34572a64fca3187bR122

If the template shows all the key value pairs for `PromlogConfig`, I am comfortable with using the struct directly. If the template cannot do that, I will split it to 2 lines.

Reply:
Done.